### DISCUSSION TITLE
css of code blocks in production not works when deployed to vercel

### DISCUSSION THREAD

**Discussion by NaviTheCoderboi** [vercel/next.js#55127] [Help]
### Summary

have made my blog using nextjs and mdx but the syntax highlighting or css of code blocks is not working when deployed to vercel,it also works when i build and server it locally

### Additional information

```js
I am using next-mdx-remote and react-syntax-highligher , css is not applying to my external component also , the importation card.
```

### Example

https://navithecoderboi.vercel.app/blog/making-a-blog-using-nextjs

**Comment by icyJoseph** [comment]
For some reason your code looks like this:

<img width="512" alt="Screenshot 2023-09-08 at 09 19 51" src="https://github.com/vercel/next.js/assets/21013447/daf3565c-02bb-4b3b-bbfa-646309084dfa">

Just plain strings... it should look like:

<img width="538" alt="Screenshot 2023-09-08 at 09 20 32" src="https://github.com/vercel/next.js/assets/21013447/686eb187-da54-4c36-8977-29c62806f906">

So, something is not right in your configuration for syntax highlighting.

**Reply by NaviTheCoderboi** [reply]
But the syntax highlighting works for my other blog

**Reply by icyJoseph** [reply]
Yeah, well, something is not correct on your setup. My example used App Router. Could you share a link to a repository or something?

**Reply by NaviTheCoderboi** [reply]
https://GitHub.com/NavitheCoderboi/portfolio

**Reply by NaviTheCoderboi** [reply]
@icyJoseph did u tried the repo link I provided?
